client/client-multi/client-c/src/gen/kaa_logging_gen.c

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include <stddef.h>
```

Comment:
This source file is auto-generated, so if the deleted "include" is needed due the compatibility reason, the right place for it is a source template.